modules/git/git.go

```diff
@@ -286,7 +286,12 @@ func syncGitConfig() (err error) {
 		}
 	}
 
-	return nil
+	if setting.Git.HTTPFollowRedirects == "" {
+		setting.Git.HTTPFollowRedirects = "false"
```

Comment:
I think we should not set it when unset in condig, so git's default (`initial`) can take over.

Reply:
…Or we set the default as `initial` as well.

Reply:
No. We have to override "initial" only explicit "initial" should be allowed.